internal/configs/virtualserver.go

```diff
@@ -586,6 +594,30 @@ func (vsc *virtualServerConfigurator) generatePolicies(owner runtime.Object, own
 					VerifyDepth:  verifyDepth,
 				}
 
+			} else if pol.Spec.EgressMTLS != nil {
+				if context != routeContext {
+					vsc.addWarningf(owner, `EgressMTLS policy is not allowed in the %v context`, context)
+					policyError = true
+					break
+				}
+				if egressMTLS != nil {
+					vsc.addWarningf(owner, "Multiple egressMTLS policies are not allowed. EgressMTLS policy %q will be ignored", key)
```

Comment:
They are not allowed in the same context. I suggest re-using the JWT message:
```suggestion
					vsc.addWarningf(owner, "Multiple egressMTLS policies in the same context is not valid. EgressMTLS policy %q will be ignored", key)
```